crates/notion-core/src/error.rs

```diff
@@ -204,6 +207,7 @@ impl NotionFail for ErrorDetails {
             ErrorDetails::DepPackageReadError { .. } => ExitCode::FileSystemError,
             ErrorDetails::DownloadToolNetworkError { .. } => ExitCode::NetworkError,
             ErrorDetails::DownloadToolNotFound { .. } => ExitCode::NoVersionMatch,
+            ErrorDetails::InvalidHookCommand { .. } => ExitCode::UnknownError,

```

Comment:
We should probably clean up these `UnknownError`s at some point; IIRC, that code was meant to only be used for "unfriendly" errors (i.e. errors not migrated to this enum).

Reply:
Agreed, the next step in this will be a quest issue to handle all of the "unknown" errors, which will include all of the calls to `.unknown()` as well as fixing up these `UnknownError` exit codes to be more descriptive.

Reply:
👍